### PR TITLE
Hmt 3 navbar

### PR DESCRIPTION
**Pull request by justin-phxm**
image

Added the navBar component

image

use these variables to test the conditional rendering.
```
const isSignedIn = true;
const hasTeam = true;
```

### CODE REVIEW

File: public/MissingProfile.svg
[the remark this answers is not in the document]
Comment (justin-phxm) [reply public/MissingProfile.svg:1]: I was thinking that if we used a social sign on library or google auth, we could import their profile picture. If we used an icon, I am not sure if that would be possible.

**Comment by ideen1** [comment]
This looks good, just the svgs need to be cropped closer to the image so there's no border.

**Review by ideen1 — CHANGES_REQUESTED** [review]
Profile SVG should have no border around it, otherwise LGTM!!

**Comment by justin-phxm** [comment]
> Profile SVG should have no border around it, otherwise LGTM!!

updated!
image

**Review by ideen1 — APPROVED** [review]

**Comment by ideen1** [comment]
Looks good! It can be merged in

**Pull request merged**